<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>SQL Insert Combiner</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::ViewRestore"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="inputs_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>SQL Input File</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Input File :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lbl_input_file">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>681</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_input_browse">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_input_status">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>751</width>
       <height>16</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_input_reload">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Reload</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="outputs_groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>781</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Insert statments to be combined : </string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sb_statements_per_batch">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>30</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>500</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Output Folder: </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lbl_output_folder">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>661</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_output_browse">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_output_status">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>751</width>
       <height>16</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>781</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Output Generation</string>
    </property>
    <widget class="QPushButton" name="btn_process">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Process</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="pb_progress">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>651</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_process_status">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>761</width>
       <height>16</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
